package.json

```diff
@@ -22,8 +23,8 @@
 	"scripts": {
 		"build": "yarn dist && vite build --config demo/vite.config.js",
 		"dev": "concurrently \"yarn watch\" \"vite --config demo/vite.config.js --port 3000\"",
-		"watch": "microbundle watch --target web --globals three=THREE --external three",
-		"dist": "microbundle --target web --globals three=THREE --external three",
+		"watch": "microbundle watch --target web --globals three=THREE --external three --no-generateTypes && node scripts/copy-definition.mjs",
+		"dist": "microbundle --target web --globals three=THREE --external three --no-generateTypes && node scripts/copy-definition.mjs",
```

Comment:
I seem to be getting the same result without `--no-generateTypes`, is that needed for something?

Reply:
In my environment, running watch or dist without `--no-generateTypes` produced index.d.ts like this

```ts
import { Pathfinding } from "./Pathfinding";
import { PathfindingHelper } from "./PathfindingHelper";
export { Pathfinding, PathfindingHelper };
```

So I turned off the default type generation, will this file not be uploaded to npm?

Reply:
I see, thank you!